webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -20,14 +20,14 @@ import {
   BULK_TASK_SEARCH_INTERVAL,
   SUBSCRIPTION_TABLE_NAME,
 } from './SubscriptionConstants';
-
 import './SubscriptionsPage.scss';
 
 class SubscriptionsPage extends Component {
   componentDidMount() {
     this.props.resetTasks();
     this.props.loadSetting('content_disconnected');
     this.props.loadSubscriptions();
+    this.props.addModal({ id: MANAGE_MANIFEST_MODAL_ID });
```

Comment:
Why is this added here and in ManageManifestModal?

Reply:
@MariaAga Before I can use `setModalOpen` or `setModalClosed` the modal must be added.  I use those in both places, so that's why `addModal` is also in both.  The `addModal` action is idempotent (meaning I can send the same action twice and it won't hurt anything).  However, if I try to `setModalOpen` or `setModalClosed` and the modal doesn't exist, I throw an error.  So I think I was just doing it to be safe.

I guess I don't need to add it in `ManageManifestModal` since it should already be added in `SubscriptionsPage`.  I will remove it from `ManageManifestModal.js`